openedx/core/djangoapps/user_api/accounts/tests/test_retirement_views.py

```diff
@@ -224,8 +224,8 @@ def test_user_can_deactivate_self(self, mock_retire_dot):
         assert not self.client.login(username=self.test_user.username, password=self.test_password)
         # make sure that an email has been sent
         assert len(mail.outbox) == 1
-        # ensure that it's been sent to the correct email address
-        assert self.test_user.email in mail.outbox[0].to
+        # ensure that it's been sent to the correct email addres
+        assert updated_user.email in mail.outbox[0].to
```

Comment:
This code checks against the *retired* email address, not the valid one?

Reply:
Fixed it now.